### PR TITLE
Fix issue with fails transactions in db

### PR DESCRIPTION
**Pull request by mauricioirace**
Removes polling and fix some payments that were not showing in 'channel details' and 'payments tab'.

### CODE REVIEW

File: src/containers/channelDetail/ChannelDetailContainer.jsx [hunk 1]
```diff
@@ -14,27 +14,44 @@ import { ALL_STATUSES } from "../../constants/uiConstants";
...
-      const actualDate = new Date();
-      let oneMonthBefore = new Date().setMonth(actualDate.getMonth() - 1);
-      const oneMonthBeforeDate = new Date(oneMonthBefore);
-      let channelData = await getChannel(
-        this.props.selectedSuggestion.tokenAddress,
-        this.props.selectedSuggestion.partnerAddress
+    const selectedSuggestion = this.getSelectedSuggestion();
+    if (selectedSuggestion) {
+      if (this.props.selectedSuggestion) {
+        sessionStorage.setItem("tempSelectedSuggestion", JSON.stringify(this.props.selectedSuggestion));
```
Comment (JONAF2103) [inline src/containers/channelDetail/ChannelDetailContainer.jsx:19]: Careful, you shouldn't mix changes, this are changes from another PR that is not merged yet.

**Review by JONAF2103 — CHANGES_REQUESTED** [review]
You should remove the mixed changes from https://github.com/rsksmart/lumino-web/pull/12, those are from the channel detail change and not from this one, be careful because it will throw conflicts on merge

**Review by JONAF2103 — APPROVED** [review]
Merge this one only when the https://github.com/rsksmart/lumino-web/pull/12 get's merged

File: src/containers/channelDetail/ChannelDetailContainer.jsx [hunk 2]
```diff
@@ -14,27 +14,44 @@ import { ALL_STATUSES } from "../../constants/uiConstants";
...
+      }
+      const channelData = await getChannel(
+        this.getSelectedSuggestion().tokenAddress,
+          this.getSelectedSuggestion().partnerAddress
```
Comment (mortelli) [inline src/containers/channelDetail/ChannelDetailContainer.jsx]: looks like there's some extra whitespace here

File: src/utils/utils.js [hunk 3]
```diff
@@ -1,8 +1,4 @@
 export const isDevelopment =() =>{
-    if (!process.env.NODE_ENV || process.env.NODE_ENV === 'development') {
-        return true;
-    } else {
-        return false;
-    }
+    return false;
```
Comment (mortelli) [inline src/utils/utils.js]: this doesn't look right

File: src/reducers/paymentsReducer.js [hunk 4]
```diff
@@ -9,7 +9,7 @@ const paymentReducer = createReducer(initialState, {
   [GET_PAYMENTS](state, action) {
     return {
       ...state,
-      payments: action.data.payments
+      payments: [...action.data.payments]
```
Comment (mortelli) [inline src/reducers/paymentsReducer.js:12]: what is the consequence of this change?

**Review by mortelli — APPROVED** [review]
some comments

File: src/reducers/paymentsReducer.js (hunk 4 above)
Comment (mauricioirace) [reply src/reducers/paymentsReducer.js:12]: It's making a copy of the array. Redux works with immutable objects in order to detect changes. There were some issues with that, so we changed this.

**Pull request merged**